cms/djangoapps/contentstore/tests/test_courseware_index.py

```diff
@@ -439,12 +483,14 @@ def _clean_course_id(self):
         """ Clean all documents from the index that have a specific course provided """
         if self.course_id:
             searcher = self.get_search_engine()
-            response = searcher.search(field_dictionary={"course": self.course_id})
+            response = searcher.search(doc_type=COURSEWARE_DOCUMENT_TYPE, field_dictionary={"course": self.course_id})
             while response["total"] > 0:
                 for item in response["results"]:
-                    searcher.remove(DOCUMENT_TYPE, item["data"]["id"])
-                    searcher.remove(DOCUMENT_TYPE, item["data"]["id"])
-                response = searcher.search(field_dictionary={"course": self.course_id})
+                    searcher.remove(COURSEWARE_DOCUMENT_TYPE, item["data"]["id"])
+                response = searcher.search(
+                    doc_type=COURSEWARE_DOCUMENT_TYPE,
+                    field_dictionary={"course": self.course_id}
+                )
```

Comment:
Generally I like to keep _test_ code more verbose and linear so that it is easy to read, but I'll admit that this one is very repetitive - will update, thanks!